spec/durationformat.html

```diff
@@ -648,10 +648,9 @@ <h1>
           }.
       </emu-alg>
 
-      <emu-clause id="sec-stringpaddingbuiltinsimpl" type="abstract operation">
+      <emu-clause id="sec-validatedurationunitstyle" type="abstract operation">
         <h1>
           ValidateDurationUnitStyle (
-            _unit_: a String,
```

Comment:
Inclusion of a seemingly-irrelevant _unit_ parameter was intentional; I expect its contents to appear in the message of any thrown exception just as it would have before being factored out of GetDurationUnitOptions in https://github.com/tc39/ecma402/pull/963/commits/20daf2630a9e187c51373e8ebdeb7ef1fcab77f9. But this is obviously too subtle... perhaps, rather than dropping the parameter, we should have a note explaining how it might be used?

Reply:
Ah gotcha, yeah a note here would be helpful for clarity. In Ladybird's implementation of these changes, I've actually included a couple of extra parameters to this AO to create useful exception messages, so I'm not sure that `unit` being treated specially by the spec is all that necessary for implementers.

Reply:
#975

Reply:
Thanks! I'll go ahead and close this PR